nrfx/samples/src/nrfx_spim_spis/advanced_non_blocking/sample.yaml

```diff
@@ -3,11 +3,17 @@ sample:
   name: nrfx_spim_spis advanced non-blocking example
 tests:
   examples.nrfx_spim_spis.advanced_non_blocking:
-    tags: spim and spis
+    tags:
+      - spim
+      - and
```

Comment:
it this `and` still needed? In previous approach it was used to specify logical AND operator, maybe with list-based approach it could be skipped?

Applies to other occurrences as well

Reply:
hmm are we sure this worked properly? @permac

Reply:
tbh i never seen `and` in use in tags. i am not sure if it even makes sense. what would be a difference between: "spim spis"  and "spim and spis"? What do you mean by "previous approach"?